### ISSUE TITLE
Error Failed to download file: Missing `url` property

### ISSUE THREAD

**Issue by joshsmithson** [TomFrankly/pipedream-notion-voice-notes#65]
I had this template working last week, now I've copied it to set up on another device and am getting the error listed in the title

I'm using Dropbox cloud storage

    at Object.downloadToTmp (file:///tmp/__pdg__/dist/code/f124a850c3f703097b5494bd1efc77777c2efa4db1b6462c7952f47e82001a2f/code/Notion-Voice-Notes.mjs:329:11)
    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
    at Object.run (file:///tmp/__pdg__/dist/code/f124a850c3f703097b5494bd1efc77777c2efa4db1b6462c7952f47e82001a2f/code/Notion-Voice-Notes.mjs:1957:5)
    at null.executeComponent (/var/task/launch_worker.js:286:22)
    at MessagePort.messageHandler (/var/task/launch_worker.js:783:28)

29/01/2024, 14:34:18
Checking that file is under 300mb...
29/01/2024, 14:34:18
File size is approximately 13.5mb.
29/01/2024, 14:34:18
File is under the size limit. Continuing...
29/01/2024, 14:34:18
Checking if the user set languages...
29/01/2024, 14:34:18
User set summary language to en.
29/01/2024, 14:34:18
ACTIVE_HANDLE
This step was still trying to run code when the step ended. Make sure you promisify callback functions and await all Promises. (Reason: FSReqCallback, Learn more: https://pipedream.com/docs/code/nodejs/async/)

Do you have any ideas on what I can try?

Many thanks,

Josh

**Comment by TomFrankly** [comment]
@joshsmithson was there an error message as well? There should have been one in a red box - if you can share that, it'd be helpful!

**Comment by joshsmithson** [comment]
@TomFrankly The only error message I noticed was in the subject "Error Failed to download file: Missing url property". I did manage to fix this by downloading your template from scratch and setting it up again.

When I got the error, I was trying to duplicate the workflow within pipedream to save some time but that ended up removing the Dropbox trigger and when I added it back and tested I got that error.
